src/components/pages/room.less

```diff
@@ -0,0 +1,9 @@
+.Room {}
+
+.Room-membersCollapse {
+  margin-top: 16px;
+}
+
+.Room-invitationsCollapse {
```

Comment:
You could make just one class ("Room-sectionCollapse" or something like this) and use for both.

Reply:
Done